packages/strapi-parts/src/TextInput/TextInput.stories.mdx

```diff
@@ -11,11 +12,33 @@ import { Tooltip } from '../Tooltip';
 
 # TextInput
 
-This is the doc of the `TextInput` component
+TextInputs are inputs that allow users to add or edit text. They support several text formats including numbers.
 
-## TextInput
+**Best practices**
 
-Description...
+- TextButton should be above - always
```

Comment:
Yeah this is not very clear 😰  I meant, if you want to add a TextButton to an input, it will have to be above that input, not on the right or left, nor below